cli/scripts/cli.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
@kakulisen yeah, it works. I think you can add a stage in travis ci to test if this script is ok, use this command cli.sh help is just fine.